app/serializers/user_serializer.rb

```diff
@@ -44,4 +50,18 @@ class UserSerializer
       CIP_TYPES[:none]
     end
   end
+
+  attributes :induction_programme_choice do |user|
+    if user.participant?
+      school_cohort = SchoolCohort.find_by(school: user.school)
```

Comment:
I think we should specify `cohort` here. You should be able to get it via 
```
  def cohort
    return early_career_teacher_profile.cohort if early_career_teacher?
    return mentor_profile.cohort if mentor?
  end
```
on the `user` model. If the `cohort` ends up `nil` it probably means our seeds don't specify cohort for a particular ECT or mentor, and we should fix that.